frontend-opal/src/components/WatchList/WatchList/WatchList.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import { View, Text } from 'react-native';
+import { AntDesign } from 'react-native-vector-icons';
+import { Button } from 'native-base';
+import PropTypes from 'prop-types';
+import styles from './styles';
+import InstrumentItems from '../InstrumentItems/instrumentItems';
+import ChartDetail from '../ChartDetail/ChartDetail';
+
+
+const WatchList = ({
+  watchlistArr, info, stockData, chartData,
+}) => (
+  <View>
+    {watchlistArr.map((item) => {
+      const [showwatchlist, setShowwatchlist] = useState(false);
```

Comment:
@martinupset let's extract this into a `WatchListItem` component

Reply:
this component should have the `info`, the `stockData` and the `chartData` inside as a constant

Reply:
ok